### PR TITLE
BOM-1107

### PR DESCRIPTION
**Pull request by awais786**
Updating to django2
Removing Toxenv variables.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,22 @@
...
 env:
 - TOXENV=django111
 - TOXENV=django20
+- TOXENV=django21
+- TOXENV=django22
 matrix:
   exclude:
   - python: 2.7
     env: TOXENV=django20
+  - python: 2.7
```
Comment (Ayub-Khan) [inline .travis.yml]: Why tox env when there is no tox file.

Comment (awais786) [reply .travis.yml]: no idea it happening in many repos now.

Comment (jmbowman) [reply .travis.yml]: These can be removed for repos like this that don't have tox configured; it just results in the same test script running in multiple Travis workers, which is a bit wasteful.

**Comment by awais786** [comment]
@jmbowman I did not find any place to test this repo. Do you have any idea where its version will get updated after release ?

**Pull request merged**